pymobiledevice3/lockdown.py

```diff
@@ -817,6 +817,10 @@ async def get_mobdev2_lockdowns(
             continue
 
         for ip in answer.ips:
+            if "tun0" in ip:
```

Comment:
Shoudn't we skip all tun interfaces instead of only tun0?

Reply:
Would you mean just simple look for string `tun`

Reply:
I don't know how to reprudce this bug, but it sounds like yeah, it should be only `tun` in case there are other existing tunnels, right?
also, i would prefix it with `%` to make it more concise